lib/sky-pages-module-generator.js

```diff
@@ -67,6 +76,11 @@ function getSource(skyAppConfig) {
     'SkyAppViewportService'
   ];
 
+  if (skyAppConfig.runtime.command === 'pact') {
+    runtimeProviders
```

Comment:
See if you can combine this with the if statement online 51, so you only have to compare the command once.